src/bikeracks/votes.py

```diff
@@ -51,120 +47,97 @@ def get_vote_status():
     if result:
         result = h.dict_from_row(result)
         return jsonify(result)
-    else:
-        return jsonify(None)
+    
             
 @votes.route('/submit_vote', methods=['POST'])
 def submit_vote():
     # insert a vote into the vote db for rack with rack_id = rack_id, vote by
     # user with user_id = user_id and vote_type=vote_type
-    if request.method != 'POST':
-        return "Bad Request", 400
-    
+
+    rack_id = request.args.get('rack_id', None, type=int)
+    user_id = request.args.get('user_id', "", type=str)
+    vote_type = request.args.get('vote_type', None, type=int)
```

Comment:
None of these default values match the `type`. You're not going to get these.

Reply:
I dropped all the default types.

Reply:
Dropped the types or dropped the default values?

Reply:
default values.